foundation-extensions/profile/profile-api-references.md

```diff
@@ -722,13 +722,13 @@ Gets the user profile attributes with the given keys.
 static func getUserAttributes(attributeNames: [String], completion: @escaping ([String: Any]?, AEPError) -> Void)
 ```
 
-* _callback_ is invoked after the customer attributes are available.
+* _completion_ the callback `closure` which will be called with user attributes.
```

Comment:
I'm not exactly sure what this is trying to communicate here - is this saying that completion is the callback that is called after `closure` is called? Is it saying that completion is the callback that closure calls?

I need further clarification on this before I can really make a suggestion here.

Reply:
This would be more accurate :
completion_ the callback `function ` which will be called with user attributes.

@calebk1 any thoughts?

Reply:
That makes a lot more sense. Though, in this case, since we're referring to function as a general thing, I think we can go with "_completion_ is the callback function which will be called with user attributes."